Open_Source_Events/Cross WOC/README.md

```diff
@@ -0,0 +1,9 @@
+<h1 align="center"> Cross Winter of Code (CrossWoC) </h1>
+
+## About CrossWoC 
+
+CrossWoC is a six-week long opensource event organised by IEEE DTU & IEEE DTU CS, which gives programmers and innovators an opportunity to bring out their nascent talent and find intriguing solutions to real-world problems. It provides a platform for developers to dig deeper into their gray matter and bring out their latent creativity through open source.This is the first edition of CrossWoC conducted by IEEE DTU & IEEE DTU CS, and it will be held from 13th February, 2020, to 28th March, 2021.
```

Comment:
Ran a Grammarly check to optimize the content: 

`
CrossWoC is a six-week-long opensource event organised by IEEE DTU & IEEE DTU CS, which allows programmers and innovators to bring out their nascent talent and find intriguing solutions to real-world problems. It provides a platform for developers to dig deeper into their grey matter and bring out their latent creativity through open source. It is the first edition of CrossWoC conducted by IEEE DTU & IEEE DTU CS, and it will be held from 13th February 2020 to 28th March 2021.
`